### ISSUE TITLE
Error in aria headings in metadata panel

### ISSUE THREAD

**Issue by jvine** [ProjectMirador/mirador#2578]
Just caught this. Issue is in here:

<img width="224" alt="Screen Shot 2019-04-19 at 11 48 37 AM" src="https://user-images.githubusercontent.com/6945691/56438839-1bf3ed80-6299-11e9-8537-bd2ac842593b.png">

Intent was for the overline + heading to be read by the screenreader like this:

> "Current item: 3"
> "Resource: Self-Portrait Dedicated to Paul Gauguin"

What we're actually getting is this:

> "Current item: Collapse current item section"
> "Resource: Collapse resource section"

and missing the actual content part of the heading.

Current HTML structure is like this:

```
<span id="overline">
overline text
<button>Collapse...</button
</span>
<h4 aria-labelledby-"overline">Canvas label etc.</h4>
```

Two issues:
1. I didn't structure the `aria-labelledby` correctly. We need to provide an ID for **both** the overline **and** the `<h4>`, and then refer to both in the `aria-labelledby`.
2. The overline text needs to be separate from the button text. 

Something like this:

```
<span>
<span  id="overline">overline text</span>
<button>Collapse...</button>
</span>
<h4 id="thisheading" aria-labelledby-"overline thisheading">Canvas label etc.</h4>
```
